app/models/requests/CreateClaimRequest.scala

```diff
@@ -91,10 +92,16 @@ object CreateClaimRequest {
       LocalDate.now(),
       payeeIndicator,
       paymentMethod,
-      declarantReNumber
+      declarantReNumber,
+      declarantName
     )
 
-    def getAgentImporterAddress(userAnswers: UserAnswers): Option[Address] = userAnswers.get(AgentImporterAddressPage) match {
+    def getDeclarantName(userAnswers: UserAnswers): Option[String] = userAnswers.get(ClaimantTypePage) match {
+      case Some(ClaimantType.Importer) => userAnswers.get(DeclarantNamePage).map(_.toString)
+      case _ => Some("DummyData")
```

Comment:
case otherwise can you replace with case Some(ClaimantType.Representative), if not importer only representative is the other option, that is why.

Reply:
Yes , I know but if you do match it with representative journey , then there will be a compiler warning for none which does not exist